tests/rbd/rbd_utils.py

```diff
@@ -962,3 +980,55 @@ def verify_migration_abort(rbd, pool, image):
         f"Image {image} is not found in pool {pool}, abort image migration is successful"
     )
     return 0
+
+
+def verify_image_feature_enable(rbd, pool, image, feature_name):
+    """Verify image feature enable.
+
+    This method will verify for the image feature is enabled on an image.
+
+    Args:
+        rbd: rbd object
+        pool: name of the pool
+        image: name of the image
+    """
+    image_load = rbd.image_info(pool, image)
+    if feature_name in image_load.get("features", {}):
+        log.info(f"Image feature {feature_name} is enabled on {image}")
+        return 0
+    log.error(f"Image feature {feature_name} is not enabled on {image}")
+    return 1
+
+
+def verify_image_feature_disable(rbd, pool, image, feature_name):
+    """Verify image feature disable.
+
+    This method will verify for the image feature is enabled on an image.
```

Comment:
It should be `disabled` on an image

Reply:
Removed this method and added  new method.